_layouts/default.html

```diff
@@ -11,7 +11,7 @@
       {% include header.html %}
       {% include nav.html %}
       <div class="main">
-        <h2 class="main-page-title">{{ page.title }}</h2>
+        <h2 class="s-h1">{{ page.title }}</h2>
```

Comment:
Yeah, thanks for bringing this up. I think the reason we hadn't (which I'm not convinced it's the right reason by any means, but just so you know what's going through my head) is, we already have the site title "Savile Design System" in an `h1` and trying to comply with the standard that there should only 1 `h1` per page. 

I just did some reading and stand by only wanting to have one `h1` but maybe the title of the site shouldn't be the `h1`, so that the actual page title can use that tag?
- [From MDN: ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) Using more than one <h1> will not result in an error, but is not considered a best practice. Using only one <h1> is beneficial for screenreader users, and SEO.
- But if the site title isn't an `h1`, what tag is appropriate to use?
- Most articles and blogs I found about this are very focused on SEO. I don't imagine that's a huge priority for this but would to follow a best practice.
- I thought [a11yproject.com](https://www.a11yproject.com/resources/) would be good to compare to since I couldn't find anyone writing about this specifically... their `h1` is the page title, and name of website is inside an svg. So maybe it's ok to put "Savile Design System" in... whatever? paragraph?

Reply:
Oh yea I don't think "Savile Design System" should be the `h1` of every page. You can put it into a `<div>` with some h1 styling if we want that. The `h1` should be the page titles. Then if we did care about SEO those would be the additional links on a google search like 'Program', 'FAQs', etc. At least in theory

<img width="663" alt="Screen Shot 2021-03-29 at 7 05 25 PM" src="https://user-images.githubusercontent.com/1551316/112918258-ba15a280-90c1-11eb-9637-c1758a281532.png">